rust/src/lib/query_apply/sriov.rs

```diff
@@ -54,6 +55,17 @@ impl SrIovConfig {
                 }
             };
 
+        if let Some(autoprobe) = cur_pf_iface
+            .ethernet
+            .as_ref()
+            .and_then(|eth_conf| eth_conf.sr_iov.as_ref())
+            .and_then(|sriov_conf| sriov_conf.drivers_autoprobe.as_ref())
```

Comment:
Skip the verification if current state has `drivers_autoprobe: false` is wrong, we might pass the verification when desired `drivers_autoprobe: true` but current `drivers_autoprobe: false`.

I think you should skip sriov vf iface verification only when desired state has `drivers_autoprobe: false` or desired not mentioned with current `drivers_autoprobe: false`.